README_ADMIN.md

```diff
@@ -0,0 +1,218 @@
+can preemptively build specific content items using the `rsconnect-admin build` subcommands.
+This way the user does not have to pay the build cost when the content is accessed next.
+
+The following are some common scenarios where performing a content build might be necessary:
+
+- OS upgrade
+- changes to gcc or libc libraries
+- changes to Python or R installations
+- switching from source to binary package repositories or vice versa
+- migrating from local content execution to remote content execution with Kubernetes
+
+Administrators should use the [`rsconnect-admin content search`](#search) subcommand to help
+identify content items for building.
+
+To build a specific content item, first `add` it to the list of content that is
+"tracked" for building using its GUID.
+
+```bash
+# add a single content item by guid
+$ rsconnect-admin build add --guid 4ffc819c-065c-420c-88eb-332db1133317
+```
+
+Administrators can view currently "tracked" content items using the
+`rsconnect-admin build ls` subcommand.
+
+```bash
+# list all tracked content
+$ rsconnect-admin build ls
+
+# list tracked content that still needs to be built
+$ rsconnect-admin build ls --status NEEDS_BUILD
+
+# list tracked content that has already been built successfully
+$ rsconnect-admin build ls --status COMPLETE
+
+# list tracked content that failed to build
+$ rsconnect-admin build ls --status=ERROR
+
+# list tracked content that was cancelled while in progress
+$ rsconnect-admin build ls --status=ABORTED
+```
+
+Once the content items have been "tracked", Administrators can initiate a build
+using the `rsconnect-admin build run` subcommand. This command will attempt to
+build all "tracked" content that has the status `NEEDS_BUILD`.
+
+```bash
+# build content serially
+$ rsconnect-admin build run
+
+# build content in parallel, 5 at a time
+$ rsconnect-admin build run --parallelism 5
+```
+
+Sometimes content builds will fail. Use the `rsconnect-admin build ls` command
+to identify content items that failed to build. Then use the `rsconnect-admin build logs`
+subcommand to help identify the underlying reason for the failure.
+
+```bash
+# print latest build logs as raw text
+$ rsconnect-admin build logs --guid 4ffc819c-065c-420c-88eb-332db1133317
+
+# print latest build logs as json lines
+$ rsconnect-admin build logs -f json --guid 4ffc819c-065c-420c-88eb-332db1133317
+```
+
+The `rsconnect-admin build history` subcommand can be used to list the build
+history for a given content item.
+
+```bash
+$ rsconnect-admin build history --guid 4ffc819c-065c-420c-88eb-332db1133317
+```
+
+By default the `rsconnect-admin build logs` subcommand prints the logs of the
+most recent build. To view the build logs for a specific build, provide a
+`--task-id` to the `logs` subcommand.
+
+```bash
+$ rsconnect-admin build logs --guid 4ffc819c-065c-420c-88eb-332db1133317 --task-id GoTVLYxWkbvCo2bN
+```
+
+
+### Common Usage Examples
```

Comment:
If the upper part is just reference, then the first common usage example here could be the general add, build, ls, debug failed ones, you had described above in some ways.